src/service/layers-impl.js

```diff
@@ -74,6 +79,19 @@ function positionLt(left, top) {
   };
 }
 
+/**
+ * Stores an LayoutElement's parent layer ancestry (temporarily).
```

Comment:
Mention that this an optimization is to avoid array creation in a tight inner loop for `interateAncestry`.